Dockerfile

```diff
@@ -0,0 +1,38 @@
+FROM python:3.8 as base
+
+#install poetry
+RUN pip3 install poetry
```

Comment:
Minor, but slightly better practice to install poetry without the dependency on pip, and through [the poetry recommend route](https://python-poetry.org/docs/)